src/ai-bundle/src/AIBundle.php

```diff
@@ -319,10 +319,14 @@ private function processAgentConfig(string $name, array $config, ContainerBuilde
 
                 $tools = [];
                 foreach ($config['tools']['services'] as $tool) {
+                    if (isset($tool['agent'])) {
+                        $tool['name'] ??= $tool['agent'];
+                        $tool['service'] ??= \sprintf('ai.agent.%s', $tool['agent']);
```

Comment:
`service` can't be set, right?
```suggestion
                        $tool['service'] = \sprintf('ai.agent.%s', $tool['agent']);
```